src/components/ConfirmModal/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Heading, Modal } from '@unique-nft/ui-kit';
+import React, { FC, useCallback } from 'react';
+import styled from 'styled-components';
+import { AdditionalDark } from 'styles/colors';
+
+export interface IConfirmModalProps {
+  isVisible: boolean
+  headerText?: string
+  confirmText?: string
+  cancelText?: string
+  onConfirm: () => void
+  onCancel: () => void
+  isClosable?: boolean
+}
+
+const ConfirmModal: FC<IConfirmModalProps> = ({
+  isVisible,
+  children,
+  headerText = '',
+  confirmText = 'Ok',
+  cancelText = 'Cancel',
+  onConfirm,
+  onCancel,
+  isClosable = false
+}) => {
+  const onConfirmClick = useCallback(() => {
+    onConfirm();
+    onCancel();
```

Comment:
confrim should close modal in the first place, it shouldn't be expected to call both methods